ApplicationLibCode/Application/RiaApplication.cpp

```diff
@@ -690,9 +690,7 @@ bool RiaApplication::loadProject( const QString& projectFileName, ProjectLoadAct
     }
 
     {
-        std::vector<Rim3dView*> views;
-        m_project->allViews( views );
-
+        std::vector<Rim3dView*> views = m_project->allViews();
         for ( auto view : views )
```

Comment:
Could this be simplified further? I.e.
 for (auto view : m_project->allViews())

Reply:
Agree, I also support this suggestion.

Reply:
Fixed.